app/src/main/java/de/mytfg/app/android/api/MytfgApi.java

```diff
@@ -36,6 +40,7 @@ public static void call(String apiFunction, ApiParams params, ApiCallback callba
             callback.callback(false, null, -1, null);
         } else {
             params.doLogin();
+            MainActivity.loadingBar.setVisibility(View.VISIBLE);
```

Comment:
Eine Abfrage ob loadingBar null ist, finde ich unschön.
Aber es wäre naturlich schön  mit möglichst wenig aufwand den Progress ablesen zu können.
Die Frage ist auch, ob das in der MainActivity gut untergebracht ist oder ob das besser von jedem Modul selbst kontrolliert werden sollte.
Evtl. eine MytfgApiUi Klasse, die von MytfgApi abgeleitet die Ui Funktionen ergänzt?